### ISSUE TITLE
ReorderableList and ShowAssetPreview do not work together

### ISSUE THREAD

**Issue by orangy** [dbrizov/NaughtyAttributes#123]
Add this code to `ReorderableListTest` and try to drop a prefab into the list. 

```
		[ReorderableList]
		[ShowAssetPreview]
		public List<GameObject> prefabList;
```

It doesn't work and throws NRE.

**Comment by orangy** [comment]
Same for `LabelAttribute`:

```
		[ReorderableList]
		[Label("Struct")]
		public List<SomeStruct> structList;
```

It should display `Struct 0`, `Struct 1`, etc for element captions

**Comment by dbrizov** [comment]
I know. Unity's `CustomPropertyDrawers` target each element of arrays. `ReorderableList` however is implemented with a custom solution and it targets the arrays themselves. This is the reason why it's not working. A workaround is to have something like this:

```
[ReorderableList]
public List<PrefabWrapper> prefabList;

[Serializable]
public struct PrefabWrapper
{
    [ShowAssetPreview]
    public GameObject prefab;
}
```

I know it's not pretty, but this is the best solution I can think of.

**Comment by dbrizov** [comment]
Fixed in 693dffb1629058abd0c97be17bed4e7a11baa24e

The 2 attributes can actually be combined. The problem is that it works only with the default **width** and **height** of the **[ShowAssetPreview]** attribute
